src/main/java/org/n52/wacodis/javaps/preprocessing/GptPreprocessor.java

```diff
@@ -65,13 +74,20 @@ public GptPreprocessor(String graphFile, Map<String, String> parameters, String
         this.outputFileExtension = outputFileExtension;
         this.outputFilenamesSuffix = outputFileNameSuffix;
     }
-
+    
     @Override
-    public List<File> preprocess(Product product, String outputDirectoryPath) throws WacodisProcessingException {
+    public List<File> preprocess(Product product, String outputDirectoryPath, String epsg) throws WacodisProcessingException {
         Map<String, String> params = new HashMap();
         params.put(INPUT_KEY, product.getFileLocation().getPath());
         params.put(OUTPUT_KEY, FilenameUtils.concat(outputDirectoryPath,
                 product.getName() + this.outputFilenamesSuffix + this.outputFileExtension));
+        try {
```

Comment:
Move parameter setting for EPSG code into a HashMap that will be provided when instantiating the _GptPreprocessor_ inside _LandcoverClassificationAlgorithm_.

Reply:
Changes in GptPreprocessor reversed. Now the LandcoverClassificationAlgorithm has a HashMap with the parameters (EPSG-Code)